src/utils/ethereumUtils.ts

```diff
@@ -419,40 +447,43 @@ const deriveAccountFromWalletInput = input => {
   return deriveAccountFromMnemonic(input);
 };
 
-function getBlockExplorer(network) {
+function getBlockExplorer(network: Network) {
   switch (network) {
-    case networkTypes.mainnet:
+    case Network.mainnet:
       return 'etherscan';
-    case networkTypes.polygon:
+    case Network.polygon:
       return 'polygonScan';
-    case networkTypes.optimism:
+    case Network.optimism:
       return 'etherscan';
-    case networkTypes.arbitrum:
+    case Network.arbitrum:
       return 'arbiscan';
     default:
       return 'etherscan';
   }
 }
 
-function openAddressInBlockExplorer(address, network) {
+function openAddressInBlockExplorer(
+  address: EthereumAddress,
+  network: Network
+) {
   const etherscanHost = getEtherscanHostForNetwork(network);
   Linking.openURL(`https://${etherscanHost}/address/${address}`);
 }
 
-function openTokenEtherscanURL(address, network) {
+function openTokenEtherscanURL(address: EthereumAddress, network: Network) {
   if (!isString(address)) return;
   const etherscanHost = getEtherscanHostForNetwork(network);
   Linking.openURL(`https://${etherscanHost}/token/${address}`);
 }
 
-function openTransactionInBlockExplorer(hash, network) {
+function openTransactionInBlockExplorer(hash: string, network: Network) {
   const normalizedHash = hash.replace(/-.*/g, '');
   if (!isString(hash)) return;
   const etherscanHost = getEtherscanHostForNetwork(network);
   Linking.openURL(`https://${etherscanHost}/tx/${normalizedHash}`);
 }
 
-async function parseEthereumUrl(data) {
+async function parseEthereumUrl(data: string) {
```

Comment:
This doesn't have a return type, but I think could say `void`.

Reply:
inferred automaticall
<img width="682" alt="Screen Shot 2021-12-08 at 3 12 26 PM" src="https://user-images.githubusercontent.com/1247834/145277595-d9e2f3be-b6c2-4bac-8b8f-ae47cb43cd9b.png">
y